starter-code/coffee_and_books/routes/index.js

```diff
@@ -0,0 +1,33 @@
+var express = require("express");
+var router = express.Router();
+const Place = require("../models/place");
+
+/* GET home page. */
+router.get("/", function(req, res, next) {
+  res.render("index", { title: "Express" });
+});
+
+router.post("/", (req, res, next) => {
+  // Get Params from POST
+  let location = {
+    type: "Point",
+    coordinates: [req.body.longitude, req.body.latitude]
+  };
+
+  const place = new Place({
+    name: req.body.name,
+    location: location,
+    type: req.body.place
```

Comment:
In your schema you have: name, description and location. Here you are storing: nema, location and type. The fields must match (and type is not the best option).